lua/nvim-tree.lua

```diff
@@ -407,8 +407,8 @@ local function setup_autocommands(opts)
     create_nvim_tree_autocmd("BufEnter", {
       pattern = "NvimTree_*",
       callback = function()
-        local bufnr = vim.api.nvim_get_current_buf()
         vim.schedule(function()
+          local bufnr = vim.api.nvim_get_current_buf()
```

Comment:
No need to fetch that. It's already available in callback by accepting `event` argument.